lib/Echidna.hs

```diff
@@ -53,10 +53,13 @@ prepareContract cfg fs c g = do
   ca <- view (hasLens . cryticArgs)
   si <- runSlither (NE.head fs) ca
 
+  -- filter extracted constants
+  let extractedConstants = filterConstantValue si
+
   -- load tests
   (v, w, ts) <- prepareForTest p c si
   let ads' = AbiAddress <$> v ^. env . EVM.contracts . to keys
   -- start ui and run tests
-  return (v, w, ts, Just $ mkGenDict df (extractConstants cs ++ timeConstants ++ largeConstants ++ NE.toList ads ++ ads') [] g (returnTypes cs), txs)
```

Comment:
Echidna will be quite ineffective without constant extraction in the source code, but we don't want to keep/maintein the old `extractConstant` code (since it is likely to change in future solc versions like 0.7 or beyond). Let's internally discuss this first.